examples/classification/plot_classification_probability.py

```diff
@@ -30,16 +32,39 @@
 from sklearn.metrics import accuracy_score
 from sklearn.svm import SVC
 
```

Comment:
In modern examples, we move imports to the cell where they're first used instead of leaving them on the top of the example.